src/goto-symex/goto_symex.h

```diff
@@ -164,6 +164,22 @@ class goto_symext
   /// symbolic execution from that state.
   bool should_pause_symex;
 
+  /// If this flag is set to true then assertions will be temporarily ignored
+  /// by the symbolic executions.
+  bool ignore_assertions = false;
+
+  /// \brief Defines condition for interrupting symbolic execution for a
+  ///   specific loop
+  ///
+  /// False constant by default, over-ridden by incremental BMC implementation
+  ///   to allow breaking if the unwinding the user specified loop
+  /// \param id: the loop identifier
+  /// \param unwind: current unwinding counter
+  /// \return true if the symbolic execution is to be interrupted for checking
+  virtual bool check_break(
+    const irep_idt &id,
```

Comment:
:pick: is this the loop id? given how frequent `id`s are used in this code base, I'd probably call it `loop_id`

Reply:
Done.